app/models/katello/host/content_facet.rb

```diff
@@ -88,6 +90,10 @@ def mark_cves_unchanged
         self.cves_changed = false
       end
 
+      def image_mode_host?
+        bootc_booted_image.present?
+      end
```

Comment:
May also think about adding this to scoped_search in host_managed_extensions. I would want to be able to see a list of all my image-mode hosts.

Reply:
Added!